libraries/exoplayer/src/main/java/androidx/media3/exoplayer/audio/DefaultAudioTrackBufferSizeProvider.java

```diff
@@ -62,8 +63,13 @@ public class DefaultAudioTrackBufferSizeProvider
    * devices (e.g., Broadcom 7271).
    */
   private static final int AC3_BUFFER_MULTIPLICATION_FACTOR = 2;
-
+  /**
+   * Default multiplication factor to apply to DTS Express passthrough buffer to avoid underruns
+   * on some devices (e.g., Xiaomi A2 TV).
+   */
+  private static final int DTSE_BUFFER_MULTIPLICATION_FACTOR = 4;
```

Comment:
nit: How do you think of changing the naming to `DTSHD_BUFFER_MULTIPLICATION_FACTOR`?

Reply:
Good idea. Done.